docs/development/build-instructions-windows.md

```diff
@@ -113,3 +113,8 @@ This can happen during build, when Debugging Tools for Windows has been installe
 ### ImportError: No module named win32file
 
 Make sure you have installed `pywin32` with `pip install pywin32`.
+
+### ninja stuck in endless loop of regenerating files
```

Comment:
Can you write a little more about what this looks like on the console / what causes this?

Reply:
We actually don't know what causes it 😄. Is it ok to just elaborate on how the issue looks like?